recipes/fokus/.gitignore

```diff
@@ -0,0 +1 @@
+arabic-speech-corpus
```

Comment:
I don't think we need this in 🐸TTS, as you can simply move it to somewhere else in your system.

Reply:
Oh, sorry my mistake